source/CardGames.py

```diff
@@ -219,11 +219,38 @@ def rewardPot(self, player):
 def checkIfEasterEgg(player):
   for c in player.hand:
     if(c.suit == "Hearts" and c.value == 11): #ace of hearts
-      import webbrowser
-      webbrowser.open('https://www.youtube.com/watch?v=V-_O7nl0Ii0')
       return True
   return False
 
+
+def playMiniGame(pot):
+  import random
+  secret_number = random.randint(1, 10)
+  print("I'm thinking of a number between 1 and 10. Can you guess what it is? You have 5 tries")
+  guesses = 5
+  while guesses > 0:
+    guess = int(input(str(guesses) + " guess(es): "))
+    guesses -= 1
+    if guess < secret_number:
+      print("Too low!")
+    elif guess > secret_number:
+      print("Too high!")
+    else:
+      break
+  if guess == secret_number:
```

Comment:
This game looks good. Shouldn't the if statements here be inside the while loop though? So that you can verify if the guess is correct right after they input it?

Reply:
Hi, this code checks if the guess is equal to the secret number after the while loop because the loop will break as soon as this is the case. It still has to check if they're equal, however, because one potential case is that the user inputs the wrong number too many times and runs out of guesses.